src/chat/MessageList.js

```diff
@@ -0,0 +1,31 @@
+import Component from '../Component.js';
+import MessageItem from './MessageItem.js';
+
+class MessageList extends Component {
+    render() {
+        const dom = this.renderDOM();
+
+        const messages = this.props.messages;
+        if(!messages) {
+            return dom;
+        }
+        messages.forEach(message => {
+            const messageItem = new MessageItem ({ message });
+            dom.appendChild(messageItem.render());
+        });
+        
+        setTimeout(() => {
+            dom.scrollTop = dom.scrollHeight;
+        });
+        
+        return dom;
+    }
+    renderTemplate() {
```

Comment:
maybe in the list view for messages is where the back button would go to head back to the chat list.